### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by chan227**
ready

### CODE REVIEW

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1 @@
+defaults
```
Comment (saobv-2223) [inline .browserslistrc]: add this file to .gitignore and run command: `git rm --cached .browserlistrc` to untrack it

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (saobv-2223) [inline Gemfile]: remove all the comment and sort the gem's name  A to Z please

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: in ruby file, we have to use " instead of '

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (saobv-2223) [inline config/database.yml]: You shoud create `database.yml.example` and push to git, that has same code as database.yml.
Add database.yml to gitignore and run `git rm --cached config/database.yml`.

File: test/application_system_test_case.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: We don't use this test folder in our project, so add this folder to .gitignore and run command.

**Comment by saobv-2223** [comment]
This pull request has 3 commits, so you have to merge them to ONE commit.
Notes: A pull request only has ONE commit.
commented

**Comment by saobv-2223** [comment]
commented

File: Gemfile [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
```
Comment (ngocvt-0484) [inline Gemfile]: use gem mysql2 instead of sqlite3

File: .browserslistrc (hunk 1 above)
Comment (chan227) [reply .browserslistrc]: May I know how to deal with it?

File: Gemfile (hunk 6 above)
Comment (chan227) [reply Gemfile]: So I have to delete the sqlite3 and install mysql2? How to do it?

**Comment by chan227** [comment]
> 
> 
> This pull request has 3 commits, so you have to merge them to ONE commit.
> Notes: A pull request only has ONE commit.
> commented

**Comment by chan227** [comment]
> 
> 
> This pull request has 3 commits, so you have to merge them to ONE commit.
> Notes: A pull request only has ONE commit.
> commented

File: Gemfile (hunk 6 above)
Comment (ngocvt-0484) [reply Gemfile]: remove sqlite3 and add
```suggestion
gem "mysql2"
```
and run command
bundle install

File: .browserslistrc (hunk 1 above)
Comment (saobv-2223) [reply .browserslistrc]: okie
Step 1: Open file: `.gitignore` and add this line: `.browserlistrc` (same as line 44 in bellow picture)
Step 2: Open terminal and cd to your project. Then run command: `git rm --cached .browserlistrc`
That's all !
image

Comment (chan227) [reply .browserslistrc]: Done it. Thank you.

File: Gemfile (hunk 2 above)
Comment (chan227) [reply Gemfile]: done

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (chan227) [reply app/views/layouts/application.html.erb]: done

File: config/database.yml (hunk 4 above)
Comment (chan227) [reply config/database.yml]: done

File: test/application_system_test_case.rb (hunk 5 above)
Comment (chan227) [reply test/application_system_test_case.rb]: done

File: Gemfile (hunk 6 above)
Comment (chan227) [reply Gemfile]: Done it. Thank you very much.

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

File: Gemfile [hunk 7]
```diff
@@ -0,0 +1,46 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+gem "bootsnap", ">= 1.4.2", require: false
+
```
Comment (vanvtt-0952) [inline Gemfile]: plz, remove the necessary blank lines, such as 7, 9, 11, 13, 15, 17, 19, 21, 31, 34, 40

**Comment by vanvtt-0952** [comment]
merged

**Comment by vanvtt-0952** [comment]
merged
you counld fix one my command, then you merged this pull request

File: Gemfile (hunk 7 above)
Comment (chan227) [reply Gemfile]: okay. I will try to fix it now

**Pull request merged**